### ISSUE TITLE
[snowflake] deploy error with latest sqitch

### ISSUE THREAD

**Issue by kulmam92** [sqitchers/sqitch#423]
My colleague installed sqitch today and got the below error but I can't reproduce this on my laptop.
Can you tell me where I need to check?

```
λ sqitch deploy db:snowflake://mk16660.east-us-2.azure/EDWSTG?Driver=SnowflakeDSIIDriver;warehouse=COMPUTE_WH
Adding registry tables to db:snowflake://mk16660.east-us-2.azure.snowflakecomputing.com/EDWSTG?Driver=SnowflakeDSIIDriver;warehouse=COMPUTE_WH
Got unexpected extra arguments (Files/Snowflake SnowSQL/snowsql)
Try "snowsql --help" for more information.
"c:/Program Files/Snowflake SnowSQL/snowsql" unexpectedly returned exit value 2
```

Is there a way to have sqitch output actual command it's running? Does sqitch have debug mode option?

**Comment by kulmam92** [comment]
I found the reason.
He used below command to used full path which contains white space.
`sqitch config --user engine.snowflake.client "c:/Program Files/Snowflake SnowSQL/snowsql"`

It worked after removing folder name that contains white space.
`sqitch config --user engine.snowflake.client "snowsql"`

**Comment by theory** [comment]
Yeah, that should be fixed in the next version (see notes in #413 for details).
